apps/emqx/src/emqx_keepalive.erl

```diff
@@ -39,7 +39,7 @@
 -spec(init(Interval :: non_neg_integer()) -> keepalive()).
 init(Interval) when Interval > 0 ->
     #keepalive{interval = Interval,
-               statval  = 0,
+               statval  = emqx_pd:get_counter(incoming_bytes),
```

Comment:
it's guaranteed not to be zero ?

Reply:
yes, init after Connect packet, it must be > 0 .

Reply:
Would it be better to add a new init/2 function instead of having the emqx_connection implementation coupled inside init/1?

Reply:
@HJianBo updated